src/main/java/HelloCucumber.java

```diff
@@ -0,0 +1,41 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.velocity.exception.VelocityException;
+
+import net.masterthought.cucumber.Configuration;
+import net.masterthought.cucumber.ReportBuilder;
+
+public class HelloCucumber {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [All classes and interfaces must belong to a named package](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/1879788576/issues/source?bid=3059308&fileBranchId=3088272#l11)